cloudinit/config/cloud-init-schema.json

```diff
@@ -1517,8 +1517,8 @@
       "properties": {
         "manage_etc_hosts": {
           "default": false,
-          "description": "Whether to manage /etc/hosts on the system. When set ``true``, cloud-init will render the hosts file using ``/etc/cloud/templates/hosts.tmpl`` replacing ``$hostname`` and ``$fdqn``. When set ``localhost``, Default: ``false``. DEPRECATED value ``template``will be dropped, use ``true`` instead.",
-          "oneOf": [{"type": "boolean"}, {"enum": ["template", "localhost"]}]
+          "description": "Whether to manage ``/etc/hosts`` on the system. Set ``true`` to render the hosts file using ``/etc/cloud/templates/hosts.tmpl`` replacing ``$hostname`` and ``$fdqn``. Set ``localhost`` to and cloud-init will append a ``127.0.1.1`` entry that resolves from FQDN and hostname every boot. Default: ``false``. DEPRECATED value ``template`` will be dropped, use ``true`` instead.",
```

Comment:
One more nit here: `Set localhost to and cloud-init will append`. Set localhost to what?